README.rst

```diff
@@ -0,0 +1,85 @@
+html-table-parser-python3
+=========================
+
+This module consists of just one small class.
+Its purpose is to parse HTML tables without help of external modules.
+Everything I use is part of python 3.
+You can install this module, or just copy the class located in *parse.py* into your own code.
+
+Getting Started
+---------------
+
+Installing
+~~~~~~~~~~
+
+Run pip install to this repo
+
+.. code:: bash
+
+ pip install git+git://github.com/schmijos/html-table-parser-python3.git
```

Comment:
afaik rst recognize the section even if the text indented with just one space

on this doc it mention several indentation on rst

http://docutils.sourceforge.net/docs/dev/rst/problems.html#section-structure-via-indentation

i will leave it @schmijos to choose which one and format it.

but i will change the pip install url to what  @ypid suggest